doc/development/language-server/debugging.rst

```diff
@@ -76,7 +19,7 @@ Phpactor from the shell and replay your last command:
 
 .. code:: bash
 
-   $ phpactor rpc --replay 
+   $ phpactor rpc --replay
```

Comment:
nothing in this document is specific to the language server I think, it can be moved to the parent level